packages/turf-simplify/lib/simplify.d.ts

```diff
@@ -0,0 +1,14 @@
+// Copied from https://github.com/mourner/simplify-js/blob/master/index.d.ts
```

Comment:
Can we just depend on simplify-js directly and remove our local copy?
Otherwise we probably need to preserve license text here as well.

Reply:
Included license text for now.